### PR TITLE
Add API overview page for Geometry Interfaces

### LINKED ISSUE #2647: Add API overview page for Geometry Interfaces
GroupData.json lists "Geometry Interfaces" as a Web API with the following contents:

```json
        "Geometry Interfaces": {
            "interfaces": [ "CSSMatrix",
                            "DOMMatrix",
                            "DOMMatrixReadOnly",
                            "Point" ],
            "methods":    [],
            "properties": [],
            "events":     []
        },
```

...but it doesn't have an API overview page. We should write a page and add the "overview" property to GroupData.

### PR DESCRIPTION
**Pull request by sideshowbarker**
Fixes https://github.com/mdn/content/issues/2647

### CODE REVIEW

File: files/en-us/web/api/geometry_interfaces/index.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+title: Geometry Interfaces
+slug: Web/API/Geometry_Interfaces
+page-type: web-api-overview
+tags:
+  - API
+  - CSS
+  - Geometry Interfaces
+  - Landing
+  - Reference
+  - Non-standard
```
Comment (teoli2003) [inline files/en-us/web/api/geometry_interfaces/index.md:11]:
```suggestion
  - Overview
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Yes! This was on my to-do list for months, but I never was able to get it near the top.

I'm really happy to see this PR.

One nit, and it is good to go.

File: files/jsondata/GroupData.json [hunk 2]
```diff
@@ -483,6 +483,8 @@
       "interfaces": [
```
Comment (teoli2003) [inline files/jsondata/GroupData.json:483]:
```suggestion
      "overview": ["Geometry interfaces"],
      "interfaces": [
```

File: files/en-us/web/api/geometry_interfaces/index.md (hunk 1 above)
Comment (sideshowbarker) [reply files/en-us/web/api/geometry_interfaces/index.md]: oops — thanks for catching that

File: files/en-us/web/api/geometry_interfaces/index.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+---
+title: Geometry Interfaces
```
Comment (teoli2003) [inline files/en-us/web/api/geometry_interfaces/index.md:2]:
```suggestion
title: Geometry interfaces
```

File: files/en-us/web/api/geometry_interfaces/index.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+slug: Web/API/Geometry_Interfaces
```
Comment (teoli2003) [inline files/en-us/web/api/geometry_interfaces/index.md:3]:
```suggestion
slug: Web/API/Geometry_interfaces
```

File: files/en-us/web/api/geometry_interfaces/index.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+  - api.DOMMatrix
+  - api.DOMMatrixReadOnly
+  - api.DOMPoint
+  - api.DOMQuad
+  - api.DOMRect
+  - api.DOMRectReadOnly
+---
+{{DefaultAPISidebar("Geometry Interfaces")}}
+
+**Geometry Interfaces** is a CSS module that provides interfaces for working with 3D and 2D graphics — in particular, for working with points, rectangles, quadrilaterals and [transformation matrices](/en-US/docs/Web/API/WebGL_API/Matrix_math_for_the_web#transformation_matrices) (for operations that translate/move, scale, rotate, skew/shear/slant, and flip graphics, as well as for multiplying/chaining and inverting/undoing those operations).
```
Comment (teoli2003) [inline files/en-us/web/api/geometry_interfaces/index.md:22]:
```suggestion
**Geometry interfaces** is a CSS module that provides interfaces for working with 3D and 2D graphics — in particular, for working with points, rectangles, quadrilaterals and [transformation matrices](/en-US/docs/Web/API/WebGL_API/Matrix_math_for_the_web#transformation_matrices) (for operations that translate/move, scale, rotate, skew/shear/slant, and flip graphics, as well as for multiplying/chaining and inverting/undoing those operations).
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Sorry, found some extra capitalization problems and added the overview to the `GroupData`

**Review by caugner — COMMENTED** [review]
Changes to GroupData.json LGTM.

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**